src/input_cp2k_dft.F

```diff
@@ -9721,4 +9729,52 @@ SUBROUTINE create_cdft_opt_section(section)
 
    END SUBROUTINE create_cdft_opt_section
 
+   SUBROUTINE create_dft_plus_u_section(section)
+      TYPE(section_type), POINTER                        :: section
+      TYPE(keyword_type), POINTER                        :: keyword
+
+      CPASSERT(.NOT. ASSOCIATED(section))
+
+      CALL section_create(section, __LOCATION__, &
+                          name="DFT_PLUS_U", &
+                          description="Define the parameters for the DFT+U correction implemented in the pwdft method", &
+                          n_keywords=4, &
+                          n_subsections=1, &
+                          repeats=.FALSE.)
+      NULLIFY (keyword)
+
+      CALL keyword_create(keyword, __LOCATION__, &
+                          name="_SECTION_PARAMETERS_", &
+                          description="Controls the activation of the DFT_PLUS_U section", &
+                          usage="&DFT_PLUS_U ON", &
+                          default_l_val=.FALSE., &
+                          lone_keyword_l_val=.TRUE.)
+      CALL section_add_keyword(section, keyword)
+      CALL keyword_release(keyword)
+
+      CALL keyword_create(keyword, __LOCATION__, &
+                          name="SUBSPACE_METHOD", &
+                          description="Select which subspace method to use for generating the hubbard subspace", &
+                    enum_desc=s2a("FULL_ORTHOGONALIZATION : Use all atomic wave functions to orthogonalize the hubbard subspace.", &
+                                     "ORTHOGONALIZATION : Orthogonalize the wave functions contributing to the hubbard subspace.", &
+                                        "NORMALIZATION : Normalize the wave functions contributing to the hubbard subspace.", &
+                                        "NONE : Use bare atomic wave functions "), &
```

Comment:
Please don't repeat the enum value in the description. The manual generator already takes care. Simply write:
```
enum_desc=s2a("Use all atomic wave functions to orthogonalize the hubbard subspace.", &
              "Orthogonalize the wave functions contributing to the hubbard subspace.", &
              "Normalize the wave functions contributing to the hubbard subspace.", &
              "Use bare atomic wave functions."), &
```